docs/reference/settings.md

```diff
@@ -635,11 +635,11 @@ standard, though only the following fields are respected:
 
 ### [`exclude-newer`](#exclude-newer) {: #exclude-newer }
 
-Limit candidate packages to those that were uploaded prior to the given date.
+Limit candidate packages to those that were uploaded prior to a given point in time.
 
-Accepts both [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) timestamps (e.g.,
-`2006-12-02T02:07:43Z`) and local dates in the same format (e.g., `2006-12-02`) in your
-system's configured time zone.
+Accepts [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) timestamps (e.g.,
```

Comment:
not sure if it would be useful to provide a help how to generate such a timestamp conveniently, e.g.:

```sh
date +"%Y-%m-%dT%H:%M:%S%z"
```

But probably beyond the scope of the docs.